src/dyz.c

```diff
@@ -10,6 +12,11 @@ int main(int _argc, char** _argv)
     argc = _argc;
     argv = _argv;
 
+#ifdef LUASEARCHPATH
```

Comment:
Right. The intention of this is to point to the path where the plain lua files on the target where installed and allow quick prototyping by editing them on place